roles/openshift_health_checker/openshift_checks/sdn.py

```diff
@@ -52,17 +52,26 @@ def run(self):
                 self.register_file('resolv.conf', None, '/etc/resolv.conf')
                 self.save_command_output('modules', ['/sbin/lsmod'])
                 self.save_command_output('sysctl', ['/sbin/sysctl', '-a'])
-                if self.get_var('openshift_use_crio', default=False):
+
+                use_crio = self.get_var('openshift_use_crio', default=False)
```

Comment:
`self.template_var(self.get_var('openshift_use_crio', default=False))` would be longer, but would avoid var redefinition

Reply:
I wrote it this way to avoid gnarly formatting to keep lines within 80 characters, but I can rework to avoid redefinitions if that is more important.

Reply:
I don't mind the redefine as I think it reads clearly.

Now that I'm thinking about it, we need to account for these booleans to be strings as well:
https://github.com/openshift/openshift-ansible/blob/master/roles/lib_utils/action_plugins/sanity_checks.py#L139

Reply:
That means we'll need to copy `to_bool` into this role (perhaps a class method for `OpenShiftCheck`), right? Also, we would need to expand the template first, right? Unfortunately, that means we cannot use the `convert` parameter for `get_var`.  So we need something like the following:
```python
                # ...
                use_crio = self.get_var('openshift_use_crio', default=False)
                use_crio = self.template_var(use_crio)
                if OpenShiftCheck.to_bool(use_crio):
                    # ...
```

Reply:
Yes, that would be suitable for now.